src/test/java/org/ays/auth/service/impl/AysUserUpdateServiceImplTest.java

```diff
@@ -521,4 +522,142 @@ void givenValidIdAndUserUpdateRequest_whenRolesNotExist_thenThrowAysRolesNotExis
                 .save(Mockito.any(AysUser.class));
     }
 
+
+    @Test
+    void givenValidId_whenUserIsPassive_thenActivateUser() {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+
+        // When
+        Institution mockInstitution = new InstitutionBuilder()
+                .withValidValues()
+                .build();
+        Mockito.when(identity.getInstitutionId())
+                .thenReturn(mockInstitution.getId());
+
+        AysUser mockUser = new AysUserBuilder()
+                .withValidValues()
+                .withId(mockId)
+                .withInstitution(mockInstitution)
+                .withStatus(AysUserStatus.PASSIVE)
+                .build();
+
+        Mockito.when(userReadPort.findById(Mockito.anyString()))
+                .thenReturn(Optional.of(mockUser));
+
+        Mockito.when(userSavePort.save(Mockito.any(AysUser.class)))
+                .thenReturn(Mockito.mock(AysUser.class));
+
+        // Then
+        userUpdateService.activate(mockId);
+
+        // Verify
+        Mockito.verify(userReadPort, Mockito.times(1))
+                .findById(Mockito.anyString());
+
+        Mockito.verify(userSavePort, Mockito.times(1))
+                .save(Mockito.any(AysUser.class));
+    }
+
+    @Test
+    void givenValidId_whenUserNotFound_thenThrowAysUserNotExistByIdException(){
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
```

Comment:
Testlerde verileri daha net görebilmek adına aşağıdaki şekilde kullanabiliriz, diğer testlerdeki kullanımlar da yavaş yavaş düzeltilecek
```suggestion
        String mockId = "9f1eb072-7830-4c43-9a32-d77b62ccddd3";
```